tests/parser/features/iteration/test_for_in_list.py

```diff
@@ -597,19 +597,6 @@ def test_for() -> int128:
     """,
         TypeMismatch,
     ),
-    (
-        """
-@external
-def test_for() -> int128:
```

Comment:
Was this test case removed? Why?

Reply:
The change that led to this removal was reverted. I will add this back.